tutorials/CMakeLists.txt

```diff
@@ -505,6 +505,15 @@ if(ROOT_pyroot_FOUND)
              sql/sqlfilldb.py       # same as the C++ case
              sql/sqlselect.py       # same as the C++ case
              )
+
+  if(NOT pyroot_experimental)
+    list(APPEND pyveto roofit/rf501_simultaneouspdf.py)
+    list(APPEND pyveto roofit/rf502_wspacewrite.py)
```

Comment:
There is another tutorial that depends on this one:
https://github.com/root-project/root/blob/97369539c78de8573538eb8ed727aac00b18c62d/tutorials/CMakeLists.txt#L380

Can you check if the wspaceread tutorial needs to be vetoed as well, or being dependent on a vetoed tutorial is enough to not execute?

Also, can you include a comment here on why we disable them? `Disable in old PyROOT, which lacks `Import` pythonization`.